docs/reference/scripting/security.asciidoc

```diff
@@ -90,6 +90,9 @@ script.allowed_types: inline <1>
 <1> This will allow only inline scripts to be executed but not stored scripts
 (or any other types).
 
+Setting `script.allowed_types: null` or `script.allowed_types: inline, stored`
```

Comment:
Since this is an `elasticsearch.yml` setting, I'm not sure `null` works here? In the cluster/index settings case, these are sent through as json, and null has meaning, but in yaml I think it would be a literal "null" value. I also don't see why something like that is needed? This isn't a changeable setting, it is simply something we read on startup.

Reply:
oh, woops. I jumped the gun here then. I will remove. I just wanted to remind users how to go back. So in that case, simply removing the setting.